df2gspread/gfiles.py

```diff
@@ -77,7 +77,8 @@ def get_worksheet(gc, gfile_id, wks_name, write_access=False):
             wks = spsh.worksheet(wks_name)
         else:
             if write_access == True:
-                wks = spsh.add_worksheet(wks_name, 1000, 100)
+                rows, cols = new_sheet_dimensions
+                wks = spsh.add_worksheet(wks_name, rows, cols)
```

Comment:
I think the error message by Google below are pretty clear. I guess you could wrap it in a try...except and retry it with default dimensions if the user input returns one of these errors, but I think that just obfuscates the issue. Better to let the user know an issue occurred. But feel free to add in error checking if you think it's necessary, happy to defer to you!

```
>>> d2g.upload(df, spreadsheet, wks_name, credentials=credentials, clean=False, new_sheet_dimensions=(10000000,100000000))
...
gspread.exceptions.RequestError: 400: This action would increase the number of columns in the sheet above the limit of 18278 columns.
```
```
>>> d2g.upload(df, spreadsheet, wks_name, credentials=credentials, clean=False, new_sheet_dimensions=('a','b'))
...
gspread.exceptions.RequestError: 400: [Line 1, Column 194, element gs:rowCount] The count must be specified.
```
Full traceback
```
>>> d2g.upload(df, spreadsheet, wks_name, credentials=credentials, clean=False, new_sheet_dimensions=(0,0))
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "df2gspread/df2gspread.py", line 95, in upload
    new_sheet_dimensions=new_sheet_dimensions)
  File "df2gspread/gfiles.py", line 81, in get_worksheet
    wks = spsh.add_worksheet(wks_name, *new_sheet_dimensions)
  File "/Library/Python/2.7/site-packages/gspread/models.py", line 89, in add_worksheet
    elem = self.client.post_feed(url, ElementTree.tostring(feed))
  File "/Library/Python/2.7/site-packages/gspread/client.py", line 228, in post_feed
    raise RequestError(ex.message)
gspread.exceptions.RequestError: 400: [Line 1, Column 194, element gs:rowCount] The count must be specified.